scipy/optimize/_bracket.py

```diff
@@ -246,11 +255,13 @@ def post_func_eval(x, f, work):
 
     def check_termination(work):
         stop = np.zeros_like(work.x, dtype=bool)
+        # Condition 0: initial bracket is invalid
+        stop[work.status == eim._EINPUTERR] = True
```

Comment:
A special case that I don't really want to deal with is `xl0 == xr0 == true_root` : ( Maybe we should just add a note about it?

Reply:
I haven't addressed this yet. I'd rather not deal with that case either. It looks like the documentation currently doesn't even mention that `xmin <= xl0 <  xm0 < xr0 <= xmax` is needed. I think just adding a note about that should be enough; the behavior will be exactly as documented.

Reply:
I just put things like this

```
 - ``-5`` : The initial bracket does not satisfy
              `xmin <= xl0 < xm0 < xr0 <= xmax`.
```

when documenting the error codes. Do you think that's sufficient, or should this go somewhere else as well? I was thinking of putting it in the `Parameters` documentation, but wasn't sure where to include it, since it pertains to every one of these parameters. I guess it could go in the `xm0`, stating that the parameters are defined below; or in the `xmin, `xmax` section.